main.tf

```
@@ -29,13 +29,14 @@ resource "google_compute_global_forwarding_rule" "https" {
   count      = "${var.ssl ? 1 : 0}"
   name       = "${var.name}-https"
   target     = "${google_compute_target_https_proxy.default.self_link}"
-  ip_address = "${google_compute_global_address.default.address}"
+  ip_address = "${var.ip_address}"
```

Hi

I think you forgot to handle the case when we don't provide one IP ;) 

```suggestion
  ip_address = "${var.create_ip_address ?  google_compute_global_address.default.address : var.ip_address}"
```

Thanks !